hazelcast-enterprise/Dockerfile

```
@@ -35,32 +39,44 @@ ENV HZ_HOME="${HZ_HOME}" \
     CLASSPATH="" \
     JAVA_OPTS=""
 
+LABEL name="Hazelcast IMDG Enterprise" \
+      maintainer="info@hazelcast.com" \
+      vendor="Hazelcast, Inc." \
+      version="8.1" \
+      release="${HZ_VERSION}" \
+      url="http://www.hazelcast.com" \
+      summary="Hazelcast IMDG Enterprise Image, certified to RHEL 8.1" \
```

```suggestion
      summary="Hazelcast IMDG Enterprise Image, certified to RHEL" \
```

To reduce the maintainance burden.

---

Done 👍